### DISCUSSION TITLE
Thank you, works fine

### DISCUSSION THREAD

**Discussion by ghost** [bulletmark/dyndns#2] [General]
Rpi3b, using noip.com.  Thank you.

the config file looks like this, for your information
#noip
  [[urls]]
  url = 'http://rusername:password@dynupdate.no-ip.com/nic/update?hostname=milo2atthestables.ddns.net&myip=<ipv4>'
  response = 'good <ipv4>'

**Comment by ghost** [comment]
Sorry, should have said: on bookworm

**Comment by bulletmark** [comment]
I presume this discussion is intended to be related to issue #3 which you raised at the same time? Continue any comment there please.

**Comment by ghost** [comment]
HiNo this was the first thing I posted, just to say thank you for the service.Best wishesRHOn 23 May 2024, at 01:05, Mark Blakeney ***@***.***> wrote:﻿
I presume this discussion is intended to be related to issue #3 which you raised at the same time? Continue any comment there please.
